Sources/Views/InputStackView.swift

```diff
@@ -69,4 +69,12 @@ open class InputStackView: UIStackView {
         distribution = .fill
         alignment = .bottom
     }
+    
+    open override func layoutIfNeeded() {
+        super.layoutIfNeeded()
+        
+        // We need to invalidate the size of the superview (MessageInputBar) in case the subview heights have changed
+        // and thus the superview size needs to change
+        superview?.invalidateIntrinsicContentSize()
```

Comment:
What if subview heights changed but not call `layoutIfNeeded`? (Maybe just call `setNeedsLayout`、`setNeedsUpdateConstraints `..... )

Reply:
Have you tested that? Because MessageInpurBar's intrinsicContentSize is determined by the bounds of the top and bottom InputStackViews so we just need to make sure that it readjusts when the stackView heights change.

Reply:
No , I didn't , what's my opinion is some methods can let stackview to layout , not just `layoutIfNeeded`. Would you found some situations that must do `invalidateIntrinsicContentSize` in here?

Reply:
Made a mistake its not needed I removed this